components/blocks/nav/SiteNavbar/SiteNavbar.test.ts

```diff
@@ -1,34 +1,47 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-expect-error
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { setup, $fetch, url } from '@nuxt/test-utils-edge'
+import { describe, expect, it, vi } from 'vitest'
+
+// import from 'nuxt'
+
 import { ref } from 'vue'
 import SiteNavbar from './SiteNavbar.vue'
+// import 'root/vitest.setup'
+import { fireEvent, stubbedRender } from 'root/testUtils'
 import * as apiComposables from 'root/composables/api'
 import { User } from 'root/lib/api/data-contracts'
-import { fireEvent, stubbedRender } from 'root/testUtils'
 
-vi.mock('#app', () => ({
-  useState: vi.fn(ref),
+vi.doMock(`${url('#app')}`, () => ({
+  useRuntimeConfig: () => ({
+    public: {
+      BACKEND_BASE_URL: process.env.BACKEND_BASE_URL,
+    },
+  }),
```

Comment:
Honestly I'm just lazy to look up documentation right now so: what does `doMock()` do differently than `mock()`?

Reply:
Also; can we be sure that `url('#app')` will always resolve to the right path? Like; would arguments passed into `url()` know of the aliases we set in `nuxt.config.ts`?